std_msgs/Header header

# yaw pitch roll 
geometry_msgs/Point32 ypr

# angular rate xyz
geometry_msgs/Point32 angular

# lat long altitude 64 bit
geometry_msgs/Vector3 gps

# velocity ned
geometry_msgs/Point32 ned

# accel xyz
geometry_msgs/Point32 accel

